packages/core/content-releases/admin/src/pages/tests/Releases.test.tsx

```diff
@@ -0,0 +1,92 @@
+import { lightTheme, ThemeProvider } from '@strapi/design-system';
+import { render as renderRTL, within } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { IntlProvider } from 'react-intl';
+
+import { ReleasesPage } from '../Releases';
+
+const user = userEvent.setup();
+
+const render = () => ({
+  ...renderRTL(<ReleasesPage />, {
```

Comment:
you can directly return what renderRTL returns. No need to create an object just to use spread inside it, now that you don't have the user in the object too anymore